tests/test_model_builders.py

```diff
@@ -2168,15 +2168,21 @@ def test_logreg_builder_sequential_calls():
 # on decides to disallow some of these combinations.
 @pytest.mark.parametrize(
     "estimator_skl,n_classes",
-    [
-        (
-            LogisticRegression(multi_class="ovr"),
-            3,
-        ),
-        (
-            LogisticRegression(multi_class="multinomial"),
-            2,
-        ),
+    (
+        [
+            (
+                LogisticRegression(multi_class="ovr"),
+                3,
+            ),
+            (
+                LogisticRegression(multi_class="multinomial"),
+                2,
+            ),
+        ]
+        if not sklearn_check_version("1.8")
+        else []
```

Comment:
Should it be something like this:
```suggestion
        else [(LogisticRegression(), 2)]
```
?

Reply:
I have actually added ```LogisticRegression(solver="liblinear")``` since that should trigger a case where a linear solver is used. Thanks for the suggestion, it will now be testing again.